### ISSUE TITLE
Stop using modal dialog/overlays to show test logs on sof-01.org

### ISSUE THREAD

**Issue by marc-hb** [thesofproject/sof-test#816]
At this stage this is just a poll. I have no idea about the effort required (including some backward compatibility effort to keep old links somewhat functional), so this may never happen but at the very least it may influence future web design decisions.

- Please thumb up if you would prefer test logs to show in a regular web page when you click on PASS or FAIL.

- Please thumb down if you prefer the current design that shows test logs in a modal dialog, hiding the big PASS/FAIL overview array.

Example: https://sof-ci.sh.intel.com/#/result/planresultdetail/8576

I will notify only a few people, please "waterfall" to your organization.

I googled "Modial dialog abuse" and found this in just a couple minutes: http://www.uxforthemasses.com/overlays/

> (1) Use overlays very sparingly
> (6) Don’t include multiple steps (in our case: multiple tabs with frequently long downloads)
> If an interaction is complex enough to require multiple steps, then it’s complex enough to warrant its own page, and to be afforded the extra space and flexibility that a page provides over an overlay.
> (7) Don’t cram in too much

Filed in sof-test for the lack of a better place.

**Comment by marc-hb** [comment]
Note the "Copy URL" button in the modal dialog = a bit of a hack so it's possible to refer to specific logs.

The main reason I find the modal dialogs very annoying in this case is because I very, very frequently want to look at _more than one thing_. For instance, I want to compare logs across several platforms. Or I want to look at logs while **not** losing sight of the overview.

**Comment by plbossart** [comment]
Agree, it's beyond annoying. I have four screens in my home office, the more things I can look at in parallel the better.

**Comment by cujomalainey** [comment]
I don't use it much, but I can definitely agree with not being able to have separate windows very annoying

**Comment by fredoh9** [comment]
Agree, don't know how easy or difficult to implement.

Basically I don't like modal window either. And I prefer the URL to have all variables/filters also. So that we can easily modify the link and get to the link.

**Comment by greg-intel** [comment]
Although I'm generally a fan of modals, there are a few issues here.  As Fred pointed out, some key URL/Link information is half-hidden.  Also, even if a link is a modal, I still feel like ctrl-clicking, or right click open in new tab, should still open the link in a new tab, which these do not.  If resolving both these issues means reworking it to not be modal, then I give my thumbs up.

**Comment by bardliao** [comment]
I prefer a regular web page, but please open it in a new tab.

**Comment by XiaoyunWu6666** [comment]
Please see demo in yongan's PR https://github.com/intel-innersource/drivers.audio.ci.sof-web/pull/60 too see whether you love the new user experience

**Comment by plbossart** [comment]
@marc-hb I think this can be closed?
